src/commands/up.rs

```diff
@@ -257,3 +256,33 @@ fn parse_env_var(s: &str) -> Result<(String, String)> {
     }
     Ok((parts[0].to_owned(), parts[1].to_owned()))
 }
+
+fn try_trigger_plugin(trigger_type: &str) -> Result<String> {
+    use crate::commands::plugins::PluginCompatibility;
+    use spin_plugins::manager::PluginManager;
+
+    let subcommand = format!("trigger-{trigger_type}");
+    let plugin_manager = PluginManager::try_default()?;
+    let plugin_store = plugin_manager.store();
+    let is_installed = plugin_store
+        .installed_manifests()?
```

Comment:
This doesn't seem to actually be fallible...but in case it ever becomes fallible: `.context`?

Reply:
I'm proposing to change both this and the catalogue one so that, rather than printing error traces, they fall through to the "no plugin found" message.